### PR TITLE
[Merged by Bors] - chore(Tactic): reduce use of autoImplicit, part 3

### PR DESCRIPTION
**Pull request by grunweg**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Tactic/Polyrith.lean [hunk 1]
```diff
@@ -244,7 +242,7 @@ def Poly.pow' : ℕ → ℕ → Poly
   | i, k => .pow (.var i) (.const k)
 
 /-- Constructs a sum from a monadic function supplying the monomials. -/
-def Poly.sumM [Monad m] (a : Array α) (f : α → m Poly) : m Poly :=
+def Poly.sumM  {m : Type → Type*} {α : Type*} [Monad m] (a : Array α) (f : α → m Poly) : m Poly :=
```
Comment (Ruben-VandeVelde) [inline Mathlib/Tactic/Polyrith.lean:245]:
```suggestion
def Poly.sumM {m : Type → Type*} {α : Type*} [Monad m] (a : Array α) (f : α → m Poly) : m Poly :=
```

**Review by Ruben-VandeVelde — APPROVED** [review]
maintainer delegate

**Comment by grunweg** [comment]
On behalf of Ruben, let me
maintainer merge

**Comment by semorrison** [comment]
bors merge

**Pull request closed without merging**